Prox/Prox/PlaceDetails/PlaceDetailsCardView.swift

```diff
@@ -271,6 +274,10 @@ class PlaceDetailsCardView: UIView {
 
         PlaceUtilities.updateReviewUI(fromProvider: place.yelpProvider, onView: yelpReviewView)
         PlaceUtilities.updateReviewUI(fromProvider: place.tripAdvisorProvider, onView: tripAdvisorReviewView)
+
+        let collapsed = (place.totalReviewCount == 0)
+        reviewViewContainer.isHidden = collapsed
```

Comment:
`height == 0` will set the container constraints so that the top and bottom are the same, but subviews (images) might still be visible. So `isHidden` makes the view disappear, and `height == 0` collapses the layout.